tabstrip/DynamicTabs/readme.md

```diff
@@ -29,7 +29,7 @@ Comments in the code offer more details.
 You can dynamically create `TabStripTab` instances by looping through a collection of descriptors. To extract the underlying model from the currently active tab you can:
 
 1. Create a collection of all Visible tabs
-    * This example shows how to do that by using the `Visible` property from the `TabDescriptor` as a condition.
+    * This example shows how to do that by using the `Visible` property from the `TabDescriptor` as a condition. **As of version 2.30.0 of Telerik UI for Blazor, the `TabStripTab` exposes a `Visible` parameter which allows you to easily control the tabs visibility. Read more in [Tabs Configuration](https://docs.telerik.com/blazor-ui/components/tabstrip/tabs-configuration) article.**
```

Comment:
```suggestion
    * This example shows how to do that by using the `Visible` property from the `TabDescriptor` as a condition. **Since UI for Blazor 2.30, the `TabStripTab` exposes a `Visible` parameter. Read more at [Tabs Configuration](https://docs.telerik.com/blazor-ui/components/tabstrip/tabs-configuration)**
```